*** testsBoth/A-03-03-M-ok.out	2025-10-15 15:38:54.000000000 +0100
--- testsBoth/A-03-03-M-ok.outhyp	2025-10-15 16:11:16.396938200 +0100
***************
*** 16,23 ****
  5 - Alterar inventário de uma obra
  0 - Sair
  Escolha uma opção: 3
- Introduza o termo de pesquisa: Mest
- 2 - 3 de 3 - Livro - Analise Matematica sem Mestre - 19 - Técnica e Científica - Carlos Fonseca - 1234567890
  Menu de obras
  1 - Mostrar obra
  2 - Mostrar todas as obras
--- 16,21 ----
***************
*** 25,30 ****
--- 23,30 ----
  4 - Mostrar todas as obras de um criador
  5 - Alterar inventário de uma obra
  0 - Sair
+ Escolha uma opção: Mest
+ Número inválido!
  Escolha uma opção: 0
  Menu Principal
  1 - Abrir ficheiro
