*** testsBoth/A-07-09-M-ok.out	2025-10-15 15:38:54.000000000 +0100
--- testsBoth/A-07-09-M-ok.outhyp	2025-10-15 16:11:23.157205400 +0100
***************
*** 24,46 ****
  2 - Devolver obra
  0 - Sair
  Escolha uma opção: 2
! Introduza o número de utente: 1
! Introduza o número da obra: 3
  Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
  Escolha uma opção: 2
! Introduza o número de utente: 2
! Introduza o número da obra: 8
  Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
  Escolha uma opção: 2
! Introduza o número de utente: 2
! Introduza o número da obra: 8
! Devolver obra: Operação inválida: A obra 8 não foi requisitada pelo utente 2.
  Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
--- 24,72 ----
  2 - Devolver obra
  0 - Sair
  Escolha uma opção: 2
! Menu de Gestão de Requisições
! 1 - Requisitar obra
! 2 - Devolver obra
! 0 - Sair
! Escolha uma opção: 1
! Menu de Gestão de Requisições
! 1 - Requisitar obra
! 2 - Devolver obra
! 0 - Sair
! Escolha uma opção: 3
! Opção inválida!
! Menu de Gestão de Requisições
! 1 - Requisitar obra
! 2 - Devolver obra
! 0 - Sair
! Escolha uma opção: 2
  Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
  Escolha uma opção: 2
! Menu de Gestão de Requisições
! 1 - Requisitar obra
! 2 - Devolver obra
! 0 - Sair
! Escolha uma opção: 8
! Opção inválida!
  Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
  Escolha uma opção: 2
! Menu de Gestão de Requisições
! 1 - Requisitar obra
! 2 - Devolver obra
! 0 - Sair
! Escolha uma opção: 2
! Menu de Gestão de Requisições
! 1 - Requisitar obra
! 2 - Devolver obra
! 0 - Sair
! Escolha uma opção: 8
! Opção inválida!
  Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
