*** testsBoth/A-11-06-M-ok.out	2025-10-15 15:38:54.000000000 +0100
--- testsBoth/A-11-06-M-ok.outhyp	2025-10-15 16:11:30.447273900 +0100
***************
*** 13,73 ****
  2 - Devolver obra
  0 - Sair
  Escolha uma opção: 1
- Introduza o número de utente: 2
- Introduza o número da obra: 2
- A obra 2 deve ser devolvida até ao dia 9.
  Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
  Escolha uma opção: 2
! Introduza o número de utente: 2
! Introduza o número da obra: 2
  Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
  Escolha uma opção: 1
- Introduza o número de utente: 2
- Introduza o número da obra: 2
- A obra 2 deve ser devolvida até ao dia 9.
  Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
  Escolha uma opção: 2
! Introduza o número de utente: 2
! Introduza o número da obra: 2
  Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
  Escolha uma opção: 1
- Introduza o número de utente: 2
- Introduza o número da obra: 2
- A obra 2 deve ser devolvida até ao dia 9.
  Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
  Escolha uma opção: 2
! Introduza o número de utente: 2
! Introduza o número da obra: 2
  Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
  Escolha uma opção: 1
- Introduza o número de utente: 2
- Introduza o número da obra: 2
- A obra 2 deve ser devolvida até ao dia 9.
  Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
  Escolha uma opção: 2
! Introduza o número de utente: 2
! Introduza o número da obra: 2
  Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
--- 13,133 ----
  2 - Devolver obra
  0 - Sair
  Escolha uma opção: 1
  Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
  Escolha uma opção: 2
! Menu de Gestão de Requisições
! 1 - Requisitar obra
! 2 - Devolver obra
! 0 - Sair
! Escolha uma opção: 2
! Menu de Gestão de Requisições
! 1 - Requisitar obra
! 2 - Devolver obra
! 0 - Sair
! Escolha uma opção: 2
! Menu de Gestão de Requisições
! 1 - Requisitar obra
! 2 - Devolver obra
! 0 - Sair
! Escolha uma opção: 2
! Menu de Gestão de Requisições
! 1 - Requisitar obra
! 2 - Devolver obra
! 0 - Sair
! Escolha uma opção: 2
  Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
  Escolha uma opção: 1
  Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
  Escolha uma opção: 2
! Menu de Gestão de Requisições
! 1 - Requisitar obra
! 2 - Devolver obra
! 0 - Sair
! Escolha uma opção: 2
! Menu de Gestão de Requisições
! 1 - Requisitar obra
! 2 - Devolver obra
! 0 - Sair
! Escolha uma opção: 2
! Menu de Gestão de Requisições
! 1 - Requisitar obra
! 2 - Devolver obra
! 0 - Sair
! Escolha uma opção: 2
! Menu de Gestão de Requisições
! 1 - Requisitar obra
! 2 - Devolver obra
! 0 - Sair
! Escolha uma opção: 2
  Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
  Escolha uma opção: 1
  Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
  Escolha uma opção: 2
! Menu de Gestão de Requisições
! 1 - Requisitar obra
! 2 - Devolver obra
! 0 - Sair
! Escolha uma opção: 2
! Menu de Gestão de Requisições
! 1 - Requisitar obra
! 2 - Devolver obra
! 0 - Sair
! Escolha uma opção: 2
! Menu de Gestão de Requisições
! 1 - Requisitar obra
! 2 - Devolver obra
! 0 - Sair
! Escolha uma opção: 2
! Menu de Gestão de Requisições
! 1 - Requisitar obra
! 2 - Devolver obra
! 0 - Sair
! Escolha uma opção: 2
  Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
  Escolha uma opção: 1
  Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
  0 - Sair
  Escolha uma opção: 2
! Menu de Gestão de Requisições
! 1 - Requisitar obra
! 2 - Devolver obra
! 0 - Sair
! Escolha uma opção: 2
! Menu de Gestão de Requisições
! 1 - Requisitar obra
! 2 - Devolver obra
! 0 - Sair
! Escolha uma opção: 2
! Menu de Gestão de Requisições
! 1 - Requisitar obra
! 2 - Devolver obra
! 0 - Sair
! Escolha uma opção: 2
! Menu de Gestão de Requisições
! 1 - Requisitar obra
! 2 - Devolver obra
! 0 - Sair
! Escolha uma opção: 2
  Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
***************
*** 116,124 ****
  2 - Devolver obra
  0 - Sair
  Escolha uma opção: 1
! Introduza o número de utente: 2
! Introduza o número da obra: 2
! A obra 2 deve ser devolvida até ao dia 9.
  Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
--- 176,191 ----
  2 - Devolver obra
  0 - Sair
  Escolha uma opção: 1
! Menu de Gestão de Requisições
! 1 - Requisitar obra
! 2 - Devolver obra
! 0 - Sair
! Escolha uma opção: 2
! Menu de Gestão de Requisições
! 1 - Requisitar obra
! 2 - Devolver obra
! 0 - Sair
! Escolha uma opção: 2
  Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
***************
*** 167,174 ****
  2 - Devolver obra
  0 - Sair
  Escolha uma opção: 2
! Introduza o número de utente: 2
! Introduza o número da obra: 2
  Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
--- 234,249 ----
  2 - Devolver obra
  0 - Sair
  Escolha uma opção: 2
! Menu de Gestão de Requisições
! 1 - Requisitar obra
! 2 - Devolver obra
! 0 - Sair
! Escolha uma opção: 2
! Menu de Gestão de Requisições
! 1 - Requisitar obra
! 2 - Devolver obra
! 0 - Sair
! Escolha uma opção: 2
  Menu de Gestão de Requisições
  1 - Requisitar obra
  2 - Devolver obra
***************
*** 193,199 ****
  0 - Sair
  Escolha uma opção: 2
  Introduza o número de utente: 2
! 2 - Luke Skywalker - last.jedi@universe.com - CUMPRIDOR - ACTIVO
  Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
--- 268,274 ----
  0 - Sair
  Escolha uma opção: 2
  Introduza o número de utente: 2
! 2 - Luke Skywalker - last.jedi@universe.com - NORMAL - ACTIVO
  Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
